rust/src/delta.rs

```diff
@@ -890,19 +913,21 @@ impl DeltaTable {
     }
 
     /// List files no longer referenced by a Delta table and are older than the retention threshold.
-    fn get_stale_files(&self, retention_hours: u64) -> Result<HashSet<&str>, DeltaTableError> {
-        if retention_hours < 168 {
+    fn get_stale_files(
+        &self,
+        retention_hours: Option<u64>,
+    ) -> Result<HashSet<&str>, DeltaTableError> {
+        let delete_before_timestamp = retention_hours
+            .map(|h| Utc::now().timestamp_millis() - 3600000 * h as i64)
```

Comment:
I would suggest to extract `Utc::now().timestamp_millis() - 3600000 * h as i64` into a variable (with an appropriate name) to make reading easier.

Reply:
ok

Reply:
hmm, well, `h` is depended on the option. Do you suggest to move it to the helper function instead?